packages/core/test/integration/data-types/data-types.test.ts

```diff
@@ -1229,6 +1229,90 @@ describe('DataTypes', () => {
     });
   });
 
+  describe('UUID default values', () => {
+    beforeAll2(async () => {
+      if (dialect.name === 'postgres') {
+        await sequelize.query('CREATE EXTENSION IF NOT EXISTS "uuid-ossp"');
```

Comment:
Is this packaged by default in the Docker image we use?

Reply:
Seems to be. I haven't encountered a postgres distribution that didn't include it